### PR TITLE
Umbraco Commerce RC1 Release Notes

### PR DESCRIPTION
**Pull request by mattbrailsford**
## Description

Umbraco Commerce RC1 Release Notes

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Commerce RC1 

## Deadline (if relevant)

ASAP

### CODE REVIEW

File: 14/umbraco-commerce/release-notes/README.md [hunk 1]
```diff
@@ -17,6 +17,26 @@ If you are upgrading to a new major version, check the breaking changes in the [
...
 This section contains the release notes for Umbraco Commerce 14 including all changes for this version.
 
+#### 14.0.0-rc2 (29th July 2024)
+
+* Fixed issue in rc1 DB migration script.
+
+#### 14.0.0-rc1 (29th July 2024)
+
+* Added sort modals to all sortable entities.
+* Added simple text search across all entity collection views.
```
Comment (eshanrnh) [inline 14/umbraco-commerce/release-notes/README.md:27]:
```suggestion
* Added text search across all entity collection views.
```

File: 14/umbraco-commerce/release-notes/README.md [hunk 2]
```diff
@@ -17,6 +17,26 @@ If you are upgrading to a new major version, check the breaking changes in the [
...
+* Added cart status filter option to carts collection view.
+* Added advanced properties support to payment / shipping provider settings.
+* Added order activity log funcationality back in.
+* Added commerce section dashboard.
+* Added async pipeline implementation.
+* Added front end NPM package for UI extension points.
+* Localized all hard coded strings.
+* Replaced Newtonsoft.Json with System.Text.Json throughout.
```
Comment (eshanrnh) [inline 14/umbraco-commerce/release-notes/README.md:35]:
```suggestion
* Replaced `Newtonsoft.Json` with `System.Text.Json` throughout.
```

File: 14/umbraco-commerce/release-notes/README.md [hunk 3]
```diff
@@ -17,6 +17,26 @@ If you are upgrading to a new major version, check the breaking changes in the [
...
+* Removed ExchangeRateHost as the default exchange rate service as it now requires an API key.
```
Comment (eshanrnh) [inline 14/umbraco-commerce/release-notes/README.md:36]:
```suggestion
* Removed `ExchangeRateHost` as the default exchange rate service as it now requires an API key.
```

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @mattbrailsford 🙌 Is it okay if I remove simple from the text to fix the vale error?

File: 14/umbraco-commerce/release-notes/README.md (hunk 1 above)
Comment (mattbrailsford) [reply 14/umbraco-commerce/release-notes/README.md:27]: Can we change it to `basic`?

Comment (eshanrnh) [reply 14/umbraco-commerce/release-notes/README.md:27]: Sure, will change it to basic text search.

File: 14/umbraco-commerce/release-notes/README.md [hunk 4]
```diff
@@ -17,6 +17,26 @@ If you are upgrading to a new major version, check the breaking changes in the [
...
+#### 14.0.0-rc2 (29th July 2024)
+
+* Fixed issue in rc1 DB migration script.
+
+#### 14.0.0-rc1 (29th July 2024)
+
+* Added sort modals to all sortable entities.
+* Added basic text search across all entity collection views.
+* Added cart status filter option to carts collection view.
+* Added advanced properties support to payment / shipping provider settings.
```
Comment (eshanrnh) [inline 14/umbraco-commerce/release-notes/README.md:29]:
```suggestion
* Added advanced properties support to payment/shipping provider settings.
```

**Pull request merged**